components/list/image/src/index.scss

```diff
@@ -1,58 +1,62 @@
 @import '~@schibstedspain/sui-theme/lib/index';
 @import '~@schibstedspain/sui-image-lazy-load/lib/index';
-@import 'settings';
+@import './settings';
 
 .sui-ListImage {
   @include sui-list('custom');
-  @include media-breakpoint-only(xl) {
-    margin: -($m-v-list-image-item-half-gutter);
-  }
-  @include media-breakpoint-only(l) {
-    margin: (-($m-v-list-image-item-half-gutter)) (-($m-h-list-image-item-breakpoint-l-gutter));
-  }
-  @include media-breakpoint-down(m) {
-    justify-content: space-between;
+  @include media-breakpoint-up(l) {
+    justify-content: flex-start;
```

Comment:
Setting `justify-content` to its default value of `flex-start` is redundant.
I mean... Items inside a Flex container are using a justify-content flex-start alignment by default so you don't have to assign that value.